apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/utils/EntityManagerUtilTest.java

```diff
@@ -0,0 +1,30 @@
+package com.ctrip.framework.apollo.biz.utils;
+
+import javax.persistence.EntityManagerFactory;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * @author Ayush Jha
+ */
+
+@RunWith(MockitoJUnitRunner.class)
+public class EntityManagerUtilTest {
+
+	@InjectMocks
+	private EntityManagerUtil entityManagerUtil;
+
+	@Mock
+	private EntityManagerFactory entityManagerFactory;
+
+	@Test
+	public void testCloseEntityManager() {
```

Comment:
Is this test not finished?

Reply:
It's finished in the sense that I have covered all the lines for that class except for a couple of lines.  So the test coverage will surely increase.

Reply:
I'll fix the rest also I just committed that so that you guys can review it and if it's according to your standards then I'll move on to the next classes also, over the weekend.

Reply:
Hi can you please delete my 17th march commits from pr.

Reply:
We could not delete your commits from the pr, but you could do it yourself by following the [contribution guide](https://github.com/apolloconfig/apollo/blob/master/CONTRIBUTING.md) to `squash commits for one feature into one commit`.

BTW, as for the unit test, the purpose to increase the coverage is to improve the code quality, not just a coverage number. So it's necessary to verify the logic inside the test.

Reply:
Yeah, I'll make sure to do that in the Entity class. The other one that I've committed covers the logic too.

Reply:
So I am deleting all those commits except for that one class.